models/models.js

```diff
@@ -4,4 +4,32 @@ exports.getAllTopics = () => {
     return db.query("SELECT * FROM topics;").then(((result) => {
         return result.rows;
     }))
-} 
\ No newline at end of file
+} 
+
+exports.getAllArticles = () => {
+    const articlesData = db.query("SELECT * FROM articles ORDER BY created_at DESC")

```

Comment:
You can achieve this logic with a single sql query. An analogous example can be found in the notes [here](https://notes.northcoders.com/courses/js-back-end/sql-relationships). See the section `Group By and Aggregate Functions
`